frontend/src/pages/modelRegistrySettings/ModelRegistryTableRowStatus.tsx

```diff
@@ -118,8 +117,24 @@ export const ModelRegistryTableRowStatus: React.FC<ModelRegistryTableRowStatusPr
     }
   }
 
+  const isClickable =
+    statusLabel === ModelRegistryStatusLabel.Unavailable ||
+    statusLabel === ModelRegistryStatusLabel.Degrading;
+
   const label = (
-    <Label data-testid="model-registry-label" icon={icon} color={color} isCompact>
+    <Label
+      {...(isClickable
+        ? {
+            onClick: () => {
```

Comment:
not sure if we should prefer this over ` style={{ cursor: 'pointer' }}`, inspiration from https://github.com/opendatahub-io/odh-dashboard/pull/3344 - let me know if this works though.

Reply:
I tried to add that style but it's not changing anything from what I see - it remains the pointing hand ( like in the picture but just white and slightly bent)
<img width="61" alt="Screenshot 2024-10-17 at 12 29 18 PM" src="https://github.com/user-attachments/assets/f022ba3f-3ef0-4f24-a7be-cba517960ead">

Reply:
I think maybe the suggestion was to avoid the noop function, but I think it's probably fine either way - inline styles are also not great 🤷 

@YuliaKrimerman can you just add more detail to the comment in that noop function, something like "Click event is handled by the Popover parent, this prop enables clickable styles in the PatternFly Label"?